### PR TITLE
docs: update porch ui installation guide for gke

### PR DESCRIPTION
**Pull request by ChristopherFry**
This pull request updates the Porch UI Installation Guide for GKE with instructions on how to create Google OAuth Client.

### CODE REVIEW

**Comment by ChristopherFry** [comment]
cc @johnbelamaric

**Comment by johnbelamaric** [comment]
LGTM

We should as we discussed move this to a kpt package. But this is start.

File: site/guides/porch-ui-installation.md [hunk 1]
```diff
@@ -22,18 +22,50 @@ This setup assumes that you have a GKE cluster up and running with porch install
...
+5. Configure an OAuth consent screen, if required
+   - For scopes, select `openid`, `auth/userinfo.email`,
+     `auth/userinfo.profile`, and `auth/cloud-platform`.
+   - Add any users that will want access to the UI if using External user type
+6. Set **Application Type** to `Web Application` with these settings:
+   - `Name`: Config as Data UI (or any name you prefer)
+   - `Authorized JavaScript origins`: http://localhost:7007
+   - `Authorized redirect URIs`:
+     http://localhost:7007/api/auth/google/handler/frame
+7. Click Create
```
Comment (droot) [inline site/guides/porch-ui-installation.md:51]: Can a user do this step using `gcloud` CLI as well ? (not for this PR, but if it is possible, then we can share that command and save these steps).

**Review by droot — APPROVED** [review]
Nice. So now users can run UI for their own org.

File: site/guides/porch-ui-installation.md (hunk 1 above)
Comment (johnbelamaric) [reply site/guides/porch-ui-installation.md:51]: No :( - I looked into it yesterday. Maybe, if they are using IAP. It's not clear to me how those things interact (IAP vs OAuth 2.0 API screens above).

**Pull request merged**